docs/examples/tf/pets_training/pet_label_map.pbtxt

```diff
@@ -0,0 +1,184 @@
+item {
```

Comment:
is this file created using the ```download_and_preprocess_dataset.sh``` script. If yes, why are we checking this file in?

Reply:
This file is used in create_pet_tf_record.py to create the TFrecords. It is needed